GiEnJul/Features/ConnectionRepository.cs

```diff
@@ -4,27 +4,27 @@
 using Serilog;
 using System.Threading.Tasks;
 
-
 namespace GiEnJul.Features
 {
     public interface IConnectionRepository
     {
-        Task<Connection> InsertOrReplaceAsync(Models.Giver giver);
-        Task<Connection> InsertOrReplaceAsync(Models.Recipient recipient);
+        Task<Connection> InsertOrReplaceAsync(Models.Giver giver, Models.Recipient recipient);
+        Task<Connection> DeleteConnectionAsync(Entities.Connection connection);
```

Comment:
If we want to stay consistent, these should return a connection model, I assume this is the connection entity.

Reply:
@Sommerli do you have any comments about this?

Reply:
@RonHaland is correct, @elisabethnetland. Return types should be Models.Connection

Reply:
As our project does not have a Models.Connection-class, the return type could be the Rowkey and Partitionkey of the created entity. This way we still have a reference to the Connection entity, without needing to handle entities in the Controller.

The current usage of the returned Connection is for keeping in case a exception is thrown, and the connection needs to be deleted. DeleteAsync could be replaced to receive a RK and a PK of the row to delete. 
![image](https://user-images.githubusercontent.com/45152025/125752630-960d2b02-51f8-4655-8eae-5b73587e8be7.png)

@RonHaland